packages/node_modules/@webex/plugin-device-manager/test/unit/spec/index.js

```diff
@@ -1,11 +1,260 @@
-/*!
- * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
- */
 
 import {assert} from '@ciscospark/test-helper-chai';
+import sinon from '@ciscospark/test-helper-sinon';
 
-describe('plugin-meetings', () => {
-  describe('meetings', () => {
-    it('passes', () => assert.isTrue(true));
+import testUtils from './testUtils';
+
+require('dotenv').config();
+
+const sparkTestUsers = require('./sparkTestUsers.js');
+
+let userSet, alice, bob;
+const startMeeting = (meeting) => {
+  meeting.on('media:ready', (media) => {
+    if (!media) {
+      return;
+    }
+    if (media.type === 'local') {
+      console.log('ready-localvideo');
+    }
+    if (media.type === 'remote') {
+      console.log('ready-remotevideo');
+    }
+    if (media.type === 'remoteShare') {
+      console.log('ready-sharevideo');
+    }
+    if (media.type === 'localShare') {
+      console.log('ready-localshare');
+    }
+  });
+  meeting.on('media:stopped', (media) => {
+    if (media.type === 'local') {
+      console.log('stopped-localvideo');
+    }
+    if (media.type === 'remote') {
+      console.log('stopped-remotevideo');
+    }
+    if (media.type === 'localShare') {
+      console.log('stopped-localshare');
+    }
+    if (media.type === 'remoteShare') {
+      console.log('stopped-sharevideo');
+    }
+  });
+};
+
+
+describe('plugin-meeting', () => {
```

Comment:
why are the automation tests being placed in the unit test dir?

Reply:
Was not sure there will be selenium automation. This is similar to mocha-integration . @mattnorris  does normal npm run test runs all the files in the test folder or just the unit tests ?